src/components/share-overlay/share-overlay.js

```diff
@@ -61,6 +62,7 @@ const ShareButton = (props: Object): React$Element<any> => {
         window.open(href, '_blank', 'width=580,height=580');
         break;
     }
+    props.player.dispatchEvent(new FakeEvent(props.player.Event.SHARE_NETWORK, {shareNetworkName: buttonType}));
```

Comment:
I would keep events related to share plugin inside the plugin repo instead of put them in player